docs/1-Applying/resumes.md

```diff
@@ -78,24 +64,50 @@ Volunteer experience
 
+How do you know if you're being too verbose? If you do have a two-page resume, you should definitely at least do the exercise of shortening it until it fits into one page. Just think of it as a way to practice being concise, and ask your friends to cross out what they think is the least interesting if needed. You might not like the end result and decide not to keep it. That's fine. But you might also find out that it works just fine.
+
+It's useful to pay attention to the _information density_ of each of your sentences. How much information do I provide the reader? You can do this analysis on a word-by-word level. Here are a few examples inspired by resumes I've seen in real life.
+
+- _Worked with clients [extensively] during the planning, design, and execution phases [with the goal of producing a quality product]_. Note how the words and phrases between brackets could be removed with no loss of information. Most of the time, you can infer that you were trying to create a product
+- _Demonstrated outstanding communication skills during tutoring [by assisting with questions, teaching many efficient problem-solving methods and helping clear up concepts seen in class]_. Note how most of this sentence just defines what tutoring is.
+
+Another example is from a web design instructor position:
+- _Taught introductory web development and design classes to students [with minimal prior programming experience]_. This is implied by "introductory".
+- _Created a curriculum [with the goal of fostering creativity and better understanding of website development through web language applications]_. The key information this sentence adds is that the person not only taught a class, but designed some parts of it. However, it should be obvious that some web language had to be used and understood.
+- _Communicated ideas and concepts clearly while reviewing the main principles to build a solid foundation in web development_. This sentence provides no information. In the first half, it's expected that a teacher would communicate clearly, it's not worth mentioning unless you have concrete evidence that demonstrates the teaching quality. The second half is already part of teaching a class.
+- _Delivered lessons on using HTML, CSS and general web design principles_. This provides some information about what was taught concretely, though largely guessable.
+
+The above could be condensed into "Designed and taught an introductory class on web design using HTML and CSS. Learned how to communicate concepts clearly to students with no prior experience".
```

Comment:
I think it's worth noting that if you have very little experience, writing out some of these things a little more verbosely is okay. You do want to end up with a full page (though that page can have lots of whitespace). But being aware of which of these details the reader actually cares about is still helpful